packages/colors/docs/Component.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta, Props } from '@storybook/addon-docs';
+import { ComponentHeader, CssVars } from 'storybook/blocks';
+
+<Meta
+  title="Components/Atoms/Colors"
+  parameters={{
+    previewTabs: {
+      canvas: { hidden: true },
+    },
+  }}
+/>
+
+<!-- Docs -->
+
+<ComponentHeader
+  name="Colors"
+  stage={3}
+  design="https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Global-Design-System?node-id=1291-35195&t=WjiaeQKmFwLVgnLb-0"
+/>
+
+Colors are part of the Theme package. View [Theme Documentation](http://localhost:9009/?path=/docs/components-theme--page) for more details
```

Comment:
@esraltintas One small thing. Can you use this url so it goes to the colours section http://localhost:9009/?path=/docs/components-theme--page#colors